Classes/FormEngine/DownloadStatistics.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/***************************************************************
+ *  Copyright notice
+ *
+ *  (c) 2016 Markus Klein <markus.klein@reelworx.at>
+ *  All rights reserved
+ *
+ *  This script is part of the TYPO3 project. The TYPO3 project is
+ *  free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  The GNU General Public License can be found at
+ *  http://www.gnu.org/copyleft/gpl.html.
+ *
+ *  This script is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  This copyright notice MUST APPEAR in all copies of the script!
+ ***************************************************************/
+
+namespace BeechIt\FalSecuredownload\FormEngine;
+
+use TYPO3\CMS\Backend\Form\AbstractNode;
+use TYPO3\CMS\Core\Database\DatabaseConnection;
+use TYPO3\CMS\Lang\LanguageService;
+
+class DownloadStatistics extends AbstractNode
+{
+    /**
+     * @var array
+     */
+    protected $resultArray = array();
+
+    /**
+     * @return array
+     */
+    public function render()
+    {
+        $this->resultArray = $this->initializeResultArray();
+
+        $row = $this->data['databaseRow'];
+
+        $db = $this->getDatabase();
+        $statistics = $db->exec_SELECTgetRows(
+            'sys_file.identifier, count(tx_falsecuredownload_download.file) as cnt',
```

Comment:
Wouldn't it make more sense to use `name` here? The identifier can get really long for local storage and for other storages it could be it isn't human readable at all.